# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 1d0656c6185d
# Full Hash: 1d0656c6185de116bb9cbffbcbb92319b1c07e28
# Author: alwu <alwu@mozilla.com>
# Date: 2022-11-05 21:53:50
# Regressor Bug: 1262276
# File Overlap Count: 1
# Description:
#   Bug 1262276 - part18 : add a pref to control video seamless looping. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160869
# ==============================================================================

diff -r f436cf1a01cc -r 1d0656c6185d modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Sat Nov 05 02:23:35 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Sat Nov 05 02:23:35 2022 +0000
@@ -10104,6 +10104,12 @@
   value: true
   mirror: always
 
+# Should enable this in bug1799205.
+- name: media.seamless-looping-video
+  type: RelaxedAtomicBool
+  value: @IS_NIGHTLY_BUILD@
+  mirror: always
+
 - name: media.autoplay.block-event.enabled
   type: bool
   value: false
